resources/js/components/forms/markdown.vue

```diff
@@ -308,37 +319,28 @@
       Thumbnail.openDialog();
     }
 
-    insertAtCaret(startsWith, endsWith) {
-      this.textarea.insertAtCaret(startsWith.replace(/<br>/g, "\n"), endsWith.replace(/<br>/g, "\n"), this.textarea.isSelected() ? this.textarea.getSelection() : '');
-
-      if (this.dropdown.isDropdownVisible) {
-        this.hideDropdown();
-      }
-
+    makeBold() {
+      this.editor.makeBold();
       this.updateModel();
     }
 
-    insertCitation() {
-      this.textarea.insertAtCaret('> ', '', this.textarea.getSelection().replace(/\r\n/g, "\n").replace(/\n/g, "\n> "));
-
+    makeItalics() {
+      this.editor.makeItalics();
       this.updateModel();
     }
 
-    toggleDropdown() {
-      this.dropdown.toggle();
-      this.$nextTick(() => this.search.focus());
-    }
-
-    hideDropdown() {
-      this.dropdown.toggle();
+    makeStrikeThrough() {
+      this.editor.makeStrikeThrough();
+      this.updateModel();
     }
 
     updateModel() {
-      this.input.dispatchEvent(new Event('input', {'bubbles': true}));
+      // this.input.dispatchEvent(new Event('input', {'bubbles': true}));
+      // mam bubblować event w górę? Czy to było tylko pod <textarea>?
```

Comment:
To było dla textarea. Dokładnie rzecz biorąc, to wywoływany był event `input` aby vue.js załapało zmianę treści. 

Trzeba było tak zrobić bo wstawialiśmy markdown w textarea itp (generalnie manipulowaliśmy treścią).

Reply:
Ale to nie powinno być zrobione `this.$emit('input')` z Vue? Bo rozumiem, że to po to że używałeś `v-model`? 

Ale tak czy tak, teraz to już nie ważne :D